images/dind/node/openshift-generate-node-config.sh

```diff
@@ -8,6 +8,14 @@ source /usr/local/bin/openshift-dind-lib.sh
 # Should set OPENSHIFT_NETWORK_PLUGIN
 source /data/network-plugin
 
+function lock() {
+  local config_path=$1
+
+  exec 200>"${config_path}"/.openshift-generate-node-config.exclusivelock
+
+  flock -n 200 && return 0 || return 1
```

Comment:
What does masking the return code here achieve? The only difference is non-zero return codes get aliased as 1.